scripts/check_features_invariants.py

```
@@ -1,104 +1,13 @@
-
-    print(f"Feature invariants OK: {total} features, {ga_advertised} GA+advertised, {headline_features} in headline metric")
-    return 0
+import subprocess
+from pathlib import Path
 
 
 if __name__ == "__main__":
-    raise SystemExit(main())
+    repo_root = Path(__file__).resolve().parents[1]
+    raise SystemExit(
+        subprocess.call(["cargo", "xtask", "features", "invariants"], cwd=repo_root)
```

Ruff (0.15.6)

[error] Starting a process with a partial executable path

(S607)